packages/build-tools/src/ios/pod.ts

```diff
@@ -1,12 +1,40 @@
 import path from 'path';
 
+import fs from 'fs-extra';
 import { Ios } from '@expo/eas-build-job';
 import spawn from '@expo/turtle-spawn';
 
 import { BuildContext } from '../context';
 
 export async function installPods<TJob extends Ios.Job>(ctx: BuildContext<TJob>): Promise<void> {
   const iosDir = path.join(ctx.reactNativeProjectDirectory, 'ios');
+
+  if (ctx.env.EAS_BUILD_COCOAPODS_CACHE_URL) {
+    ctx.logger.info('Running using experimental CocoaPods cache proxy...');
```

Comment:
I would drop experimental, also sentence itself sounds a bit weird to me, "running using proxy" (running what?)

If you just want to know check which builds are using a proxy you can use the env variables listed in the first phase, if that is the case I would just remove this log. If your intention was for users to see this, maybe try to rephrase that.

Reply:
Hmm, I think we don't have any logs about using NPM and Maven proxy, so I think I will probably drop it 🤔

Reply:
I thought it might be useful info for the users in case the `Install pods` step fails after our update, so they understand better what has changed, although it might be better to keep it the same way as we did for previous ones 🤔. We are already very cautious and will only use it for 1/3 of the builds.